pkgs/applications/misc/mission-center/default.nix

```diff
@@ -102,14 +102,24 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
-    echo -e "[wrap-file]\ndirectory = nvtop-src\n[provide]\ndependency_names = nvtop" > ./subprojects/nvtop.wrap
-    cp -r --no-preserve=mode,ownership "${nvtop}" ./subprojects/nvtop-src
-    cd ./subprojects/nvtop-src
+    SRC_GATHERER=/build/source/src/sys_info_v2/gatherer
```

Comment:
Does `$NIX_BUILD_TOP` work here?

Reply:
Seems like it work. Is it what you were suggesting ?